src/Simplify_Add.cpp

```diff
@@ -188,6 +188,9 @@ Expr Simplify::visit(const Add *op, ExprInfo *bounds) {
                rewrite(x + ((c0 - x)/c1 + y)*c1, y * c1 - ((c0 - x) % c1) + c0, c1 > 0) ||
                rewrite(x + (y + (c0 - x)/c1)*c1, y * c1 - ((c0 - x) % c1) + c0, c1 > 0) ||
 
+               rewrite((x/c0) + (y*(x/c1)), (y*(x/c1))*fold((c1/c0) + 1), (c1 % c0) == 0) ||
```

Comment:
These ones fail to verify:
Incorrect rule: rewrite(((x/c0) + (y*(x/c1))), ((y*(x/c1))*((c1/c0) + 1)), ((c1 % c0) == 0))
Counterexample is: c0 = -6908
c1 = 6908
x = -6908
y = 2
For counterexample, LHS = -1 RHS = 0
Incorrect rule: rewrite(((x/c0) + ((x/c1)*y)), (((x/c1)*y)*((c1/c0) + 1)), ((c1 % c0) == 0))
Counterexample is: c0 = -6908
c1 = 6908
x = -6908
y = 2
For counterexample, LHS = -1 RHS = 0

Reply:
Doesn't seem to be related to rounding:
```
Incorrect rule: rewrite(((x/c0) + ((x/c1)*y)), (((x/c1)*y)*((c1/c0) + 1)), (((c1 % c0) == 0) && ((c0 > 0) && (c1 > 0))))
Counterexample is: c0 = 2
c1 = 4
x = 7
y = 1
For counterexample, LHS = 4 RHS = 3

Incorrect rule: rewrite(((x/c0) + (y*(x/c1))), ((y*(x/c1))*((c1/c0) + 1)), (((c1 % c0) == 0) && ((c0 > 0) && (c1 > 0))))
Counterexample is: c0 = 2
c1 = 4
x = 7
y = 1
For counterexample, LHS = 4 RHS = 3
```